Libraries/LibJS/Bytecode/Interpreter.cpp

```diff
@@ -3426,8 +3426,7 @@ ByteString CallConstruct::to_byte_string_impl(Bytecode::Executable const& execut
     StringBuilder builder;
     builder.appendff("CallConstruct {}, {}, {}, "sv,
```

Comment:
Now this has a `{}` to many

Reply:
The format string should just not have `sv` at the end of it, then this would raise a compiler error. Looks like a fair amount of `appendff` calls in this file errarntly have `sv`